src/routes/user.route.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const authController = require('../controllers/auth.controller.js');
+const { authMiddleware } = require('../middlewares/authMiddleware.js');
+const catchError = require('../utils/catchError.js');
+
+const userRouter = new express.Router();
+
+userRouter.get('/', authMiddleware, catchError(authController.activate));
```

Comment:
Ensure that `authMiddleware` is correctly implemented to call `next()` and allow the middleware chain to continue. This is crucial for the route to function correctly.